spec/tasks/javascript_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Javascript rake tasks" do
+
+  test_git_hash = 'abcdef1234567890abcdef1234567890abcdef12'
+
+  # before do
+  #   Rake::Task.clear
+  #   Discourse::Application.load_tasks
+  #   StampedFiles.stubs(:git_hash).returns(test_git_hash)
+  # end
+
+  # after(:all) do
+  #   StampedFiles.cleanup(0, test_git_hash)
+  # end
+
+  describe "javascript:update" do
+    xit "creates the expected files" do
```

Comment:
we should probably remove it rather than commit it pending.

Reply:
The non-pending version of this works locally for me, but fails on the build/CI servers. Is it worth exploring why it works in one environment but not another?